### PR TITLE
Add tests for calculations

### PR DESCRIPTION
**Pull request by debitan**
Add tests for calculations file.

### CODE REVIEW

File: sow/scripts/test_calculations.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    calculate_contract_duration,
+    calculate_number_of_payments,
+    calculate_working_days,
+    calculate_contract_fee,
+    calculate_retention_fee,
+    calculate_base_monthly_payment,
+)
+
+
+def test_calculate_contract_duration():
```
Comment (elcct) [inline sow/scripts/test_calculations.py:15]: It would be good to write a little comment about what exactly you are trying to test. The function test name may not always be descriptive enough

File: sow/scripts/test_calculations.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    assert calculate_contract_duration(
+        parse('2020-01-01'), parse('2020-03-31')
+    ) == relativedelta(months=+2, days=+30)
+
+
+def test_calculate_number_of_payments():
+    assert calculate_number_of_payments(3, 0) == 3
```
Comment (elcct) [inline sow/scripts/test_calculations.py:22]: For such multiple tests may be a good idea to use `pytest.mark.parametrize` decorator and maybe check how functions will handle possible edge cases (could be in a separate test).

Comment (debitan) [reply sow/scripts/test_calculations.py:22]: have updated to use `parametrize`. I've tried to test the edge cases in the secondary tests.
